app/models/comment.rb

```diff
@@ -77,13 +77,25 @@ def mentioned_users
     User.find_all_by_username(usernames.map {|m| m[1] }).uniq
   end
 
+  def mentioned_tag_followers
```

Comment:
> Here, is this followers of the tag who were /also/ mentioned?

No, it is followers of the _tags_ that were mentioned. I can see now how it is be confusing. I wanted a similar method to `mentioned_usernames`. Will a comment on it suffice or should I change the method name?

> your code is very readable

Thank you very much :)